src/main/java/step3/domain/LottoNumber.java

```diff
@@ -0,0 +1,65 @@
+package step3.domain;
+
+import static java.text.MessageFormat.*;
+import java.util.List;
+import java.util.Objects;
+import util.Numbers;
+
+public class LottoNumber {
+
+    public static final int MINIMUM_LOTTO_NUMBER = 1;
+    public static final int MAXIMUM_LOTTO_NUMBER = 45;
+
+    private final int number;
+
+    public LottoNumber(final int number) {
+        verifyLottoNumber(number);
+
+        this.number = number;
+    }
+
+    public int check(final List<LottoNumber> lottoNumbers) {
+        if (lottoNumbers.contains(this)) {
+            return 1;
+        }
+
+        return 0;
+    }
```

Comment:
```suggestion
    public int check(final List<LottoNumber> lottoNumbers) {
        return lottoNumbers.contains(this);
    }
```

처음에 위처럼 변경가능하다고 코멘트 드리려고 했었는데요 
잘 보니 return type 이 int 더라구요 
check라는 메서드가 boolean 을 return 할 거라고 착각하게 한 것 같아요
findWinningCount 같은 명칭은 어떨까요?

Reply:
`findWinningCount` 으로 변경하겠습니다!

제안해주신 메서드명이 더 명확하게 기능을 표현해줄 것같습니다.